docs/guides-agent-app.md

```diff
@@ -0,0 +1,485 @@
+5. The address of the app (or entity) that will be able to grant that permission or revoke it (HOW).
+
+Let's revisit an annotated version of the command we ran above:
+
+```
+dao acl create
+1. <your organization name>
+2. <your agent app address> 
+3. EXECUTE_ROLE
+4. <your voting app address>
+5. <your voting app address>
+--environment aragon:rinkeby
+```
+You should see that in our case:
+
+- **1** is the name of our organization. 
+
+- **2** is our organization's Agent app -- we are managing the permissions of our Agent app by allowing the Voting app to execute actions on behalf of it.
+
+- **3** is the EXECUTE_ROLE -- The EXECUTE_ROLE is a role defined in the Agent app: it allows an entity to execute a specific action through the Agent app (transferring tokens for example).
+
+- **4** is our organization's Voting app -- we are granting this role to our Voting app, this allows it to execute actions on behalf of our Agent app.
+
+- **5** is our Voting app again -- we are assigning it as manager of the role, this allows it to grant or revoke the permissions of this role.
+
+**Note that, same as before, this command will trigger a vote in the DAO and you'll need to vote *yes* to confirm the new permissions you've granted the Voting app.**
+
+You can do this either by using the UI again or, now that you know how to get the address of your apps, by running:
+
+```
+dao exec <your organization name> <your voting app address> vote 1 true true --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+[`dao exec`](https://hack.aragon.org/docs/cli-dao-commands) is used to perform transactions in your DAO directly from the aragonCLI. It takes at least three arguments:
+
+- The first is always the name or address of the DAO you want to interact with. In our case this is our DAO's name.
+
+- The second is the address of the app where the action is being performed. In our case this is the [Voting app](https://wiki.aragon.org/dev/apps/voting/).
+
+- The third is the name of the method being executed in the app: In our case the method is [`vote`](https://wiki.aragon.org/dev/apps/voting/index.html#casting-votes).
+
+- The remaining arguments are the arguments which the method -- in our case `vote` -- will be exectuted with. We are passing in three: `1`,  `true` and  `true`.
+
+    - The first (`1`) is the id for the vote we want to interact with. This is always an integer. Vote ids start at 0 and increment by 1 each time a vote is created.
+
+    - The second (`true`) specifies which was we want to vote: `true` means yes and `false` means no.
+
+    - And the third (`true`) specifies whether the contract should check if a vote already has enough support to be executed. By executed we mean that even if everyone else voted against, the vote would still be approved. If that's the case, the vote is executed and immediately closed. `true` means check, `false` means don't check.
```

Comment:
Vote execution means that the Voting app will perform the action that was being voted on.